sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image.js

```diff
@@ -43,7 +43,7 @@ function generateTest(pixelFormat, pixelType, pathToTestRoot, prologue) {
         gl.clearColor(0,0,0,1);
         gl.clearDepth(1);
 
-        wtu.loadTexture(gl, "/resources/red-green.png", runTest);
+        wtu.loadTexture(gl, "../../resources/red-green.png", runTest);
```

Comment:
Thanks for catching that.  These two files are moved locally but not in the git commit.  Maybe for these two I accidentally used mv instead of git mv.  Add them back. I tested locally, with this, the conformance suites work fine, including the few bugs Olli pointed out after a previous merge.